Editor/EditorCore/ProBuilderEditor.cs

```diff
@@ -705,8 +707,10 @@ void OnSceneGUI(SceneView sceneView)
                     m_WasDoubleClick = false;
             }
 
-            if (m_CurrentEvent.type == EventType.MouseUp)
+            if (m_CurrentEvent.type == EventType.MouseUp && GUIUtility.hotControl == m_DefaultControl)
             {
+                GUIUtility.hotControl = 0;
+
```

Comment:
I think we will also need to relinquish the hot control on `Ignore` or `Escape` events, and anywhere else that `m_IsDragging` is unset.

Reply:
Is Escape currently cancelling a drag in probuilder?

Reply:
If it doesn't, it should 😝